scripts/generate_dummy_data.py

```diff
@@ -53,11 +54,12 @@ def generate_transactions(categories: list[Category], n=250) -> list[Transaction
 
 
 def get_engine():
-    engine = create_engine("sqlite:///finance_tracker.sqlite", echo=False)
+    DATABASE_URL = config("DATABASE_URL_DUMMY", "sqlite:///finance_tracker.sqlite")
+    engine = create_engine(DATABASE_URL, echo=False)
```

Comment:
I think echo=False is the default, if you have a DEBUG bool you could also couple it to that: `echo=DEBUG`